docs/modules/ROOT/pages/wan-replication.adoc

```diff
@@ -1,54 +1,116 @@
+** Active-Active
+
+== Set Up WAN Replication
+
+Set up WAN replication between two clusters.
+
+. Sign into the [.console]*link:{page-cloud-console}[{hazelcast-cloud} console]* and select the cluster that you want to use as a publisher cluster.
+
++
+NOTE: If you want to set up a pair of clusters in an Active-Active configuration, select either cluster.
+
+. Select *Manage Cluster* > *Settings*.
+. On the *Cluster Settings* page, select *WAN Replication* and then click *+ Create New Replication*.
+
++
+The cluster is set as the *Active* cluster. Any data updates performed on this cluster are automatically copied to the other cluster of the pair.
+
+. Either leave the default options for the following parameters, or update them to suit your use case.
+
+* *Queue Full Behavior*: Lets you decide what to do when the replication events queue is full. 
+
+** `THROW AN EXCEPTION`: Throws a `WanReplicationQueueFullException` when the queue is full. Processing of replication events stops until action is taken.
+** `THROW AN EXCEPTION ONLY IF REPLICATION ACTIVE`: As above, but this option is only triggered when both clusters connected by WAN replication are active.
+** `NONE`: Discards replication events when the queue is full. This option is only suitable for non-critical data. 
+
+* *Queue Capacity*: Sets the maximum size for the replication events queue. When this limit is reached, the *Queue Full Behavior* is triggered.
+* *Batch Size*: Sets the maximum size of replication events that are sent in a single batch.
+* *Batch Max Delay Millis*: By default, all replication events are delayed by one second (1000 milliseconds) before they are sent to the other cluster of the pair, unless they exceed the *Batch Size*.
+
+. Click *Next >* to configure the other cluster of the pair.
+. From the *Active/Passive* list, select the correct behavior for your second cluster: 
+
+* *Passive*: Receives only data updates.
+* *Active*: Sends and receives data updates. 
+
+. Click *Save*. WAN Replication is set up.
+
++
+TIP: If you want to synchronize all data between the two clusters immediately, click *Sync*. This option is useful when your're migrating data from one cluster to another. Otherwise, WAN replication is triggered when a data update is made to a publisher cluster.
+
+== Verify WAN Replication
 
-[WARNING]
-====
-If clusters are hosted on the *same* cloud provider, you can create WAN Replication between *public <-> public*, *public <-> private*, and *private <-> private* clusters.
+Check that data is successfully replicated between two clusters connected by WAN replication.
 
-If clusters are hosted on *different* cloud providers, you can create WAN Replication only between *public <-> public* clusters.
-====
+. Connect a client to your publisher cluster and write some data to a map. You can use either cluster in an Active-Active pair.
+. Open the dashboard of the other cluster of the pair and select *Manage Cluster* > *Management Center*.
```

Comment:
Why do we need to open management center to verify that? On cluster detail page of second cluster, we can see the charts of map, and can compare it with 1st cluster. Encouraging customer to open management center and force a couple of click might not be sufficient.

Reply:
Updated as per your comments.